book/src/pod-info.md

```diff
@@ -1,6 +1,6 @@
 # Pod Info on Mount
 
-**Status:** Alpha
+**Status:** Beta
```

Comment:
Add the following line below this:

The "Pod Info on Mount" feature was introduced as alpha in Kubernetes v1.12. It was promoted to beta in Kubernetes 1.13.